maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/ProceedingsService.java

```diff
@@ -39,11 +39,11 @@ public ApplicationDTO updateCrownCourt(WorkflowRequest request, RepOrderDTO repO
                 proceedingsMapper.workflowRequestToUpdateCrownCourtRequest(request.getApplicationDTO(), request.getUserDTO());
         ApiUpdateCrownCourtOutcomeResponse updateCrownCourtResponse =
                 proceedingsApiService.updateCrownCourt(apiUpdateCrownCourtRequest);
-        ApplicationDTO applicationDTO = proceedingsMapper.updateCrownCourtResponseToApplicationDto(
+        ApplicationDTO application = proceedingsMapper.updateCrownCourtResponseToApplicationDto(
```

Comment:
Should we be updating the workflow request with the returned application here? Something like:

request.setApplicationDTO(
                proceedingsMapper.updateCrownCourtResponseToApplicationDto(
                        updateCrownCourtResponse, request.getApplicationDTO()
                )
        );